app/components/StatusModal.js

```diff
@@ -23,6 +23,7 @@ export default class StatusModal extends Component {
         break
       case NOTE_STATUS.NOTE_LOAD_SUCCESS:
       case NOTE_STATUS.NOTE_DELETE_SUCCESS:
+        this.props.setNewNoteDisabled(false)
```

Comment:
Since this will call [this](https://github.com/abahmed/Deer/blob/develop/app/reducers/header.js#L9) at the end, it doesn't matter that I set it here since it will not change if it's `false` already. :man_shrugging:

Reply:
Yes, you are right! :+1: